front/src/routes/integration/all/nuki/discover-http/index.js

```diff
@@ -0,0 +1,39 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import actions from '../actions';
+import NukiPage from '../NukiPage';
+import DiscoverTab from './DiscoverTab';
+import { WEBSOCKET_MESSAGE_TYPES } from '../../../../../../../server/utils/constants';
+
+class NukiIntegration extends Component {
+  async componentWillMount() {
+    this.props.getDiscoveredNukiDevices('http');
+    this.props.getHouses();
+    this.props.getIntegrationByName('nuki');
+
+    this.props.session.dispatcher.addListener(
+      WEBSOCKET_MESSAGE_TYPES.NUKI.NEW_HTTP_DEVICE,
+      this.props.addDiscoveredDevice
+    );
+  }
+
+  componentWillUnmount() {
+    this.props.session.dispatcher.removeListener(
+      WEBSOCKET_MESSAGE_TYPES.NUKI.NEW_HTTP_DEVICE,
+      this.props.addDiscoveredDevice
+    );
+  }
+
+  render(props) {
+    return (
+      <NukiPage user={props.user}>
```

Comment:
On this page, why is there a weird line in the middle of the screen?

![Screenshot 2025-05-08 at 09 16 38](https://github.com/user-attachments/assets/5414b6a9-4f15-4472-8f02-aa675692fe64)

Both page "MQTT search" and "Web search should be consistent